prow/gerrit/adapter/adapter.go

```diff
@@ -419,14 +419,40 @@ func (c *Controller) handleInRepoConfigError(err error, instance string, change
 }
 
 // processChange creates new presubmit/postsubmit prowjobs base off the gerrit changes
-func (c *Controller) processChange(logger logrus.FieldLogger, instance string, change client.ChangeInfo) error {
+func (c *Controller) processChange(logger logrus.FieldLogger, instance string, change client.ChangeInfo, lastUpdated time.Time) error {
 	tracker := time.Now()
 	defer func() {
 		// tracker will reset in `processChange` function, and this is the time
 		// taken for a gerrit change to be processed after the last reset.
 		logger.WithField("duration(s)", time.Since(tracker).Seconds()).Debug("Time taken for gerrit change to be processed.")
 	}()
 
+	if change.Revisions != nil {
+		var hasTestOrRetestCommand bool
+		for _, message := range currentMessages(change, lastUpdated) {
+			if strings.HasPrefix(message.Message, "/test") || strings.HasPrefix(message.Message, "/retest") {
```

Comment:
this is already checked below, I don't like this but I don't have a better idea. This check is not going to work with multi-line comments, using the regex at https://github.com/kubernetes/test-infra/blob/9aadff258d20bb9fd72055c2a0621afcedb23a54/prow/pjutil/filter.go#L30 would be more reliable

Reply:
Added another regex for commands start with `/test`.